### DISCUSSION TITLE
Shoud I be concerned about effect.html in a response?

### DISCUSSION THREAD

**Discussion by polycode-nz** [livewire/livewire#5595] [1. Help] [answered]
Looking at what the responses are when emitting to a component I see a whole heap of code in effects.html.

I've read [this](https://calebporzio.com/how-livewire-works-a-deep-dive) and still not any wiser.

What I'm doing is page/component loads.

Click a button and it `livewire.emit` to the component to get some data for a select 2 instance which does a `dispatchBrowserEvent` with it. So I'm expecting to see a pretty small response.

The data portion of the response is indeed tiny. The options for the s2 aren't in there as they're not public properties of the component.

I added an incrementing public property in the components render and I can see in the response data that variable is there and is incremented but it isn't in the page as it has a wire:ignore on it - I didn't expect to see it change.

Is there something I can do to stop the response effects.html replying with 132kb of html that doesn't get used?

I tried to return an empty view if the render method has already been hit but doesn't change effects.html.

**Comment by joshhanley** [comment, marked as answer]
@polycode-nz that is because Livewire re-rendered the page in blade, so you're seeing the latest html. But you could do this, in the method where you call `dispatchBrowserEvent`, call `$this->skipRender();`. Hopefully that should do what you're after.

**Comment by polycode-nz** [comment]
@joshhanley speedy!

Boy-oh-boy I have missed the memo on that! Thanks.

**Reply by joshhanley** [reply]
@polycode-nz no worries! 🙂
